reana-ui/src/actions.js

```diff
@@ -298,11 +298,12 @@ export function fetchWorkflowLogs(id) {
   };
 }
 
-export function fetchWorkflowFiles(id, pagination) {
+export function fetchWorkflowFiles(id, pagination, search) {
   return async (dispatch) => {
     dispatch({ type: WORKFLOW_FILES_FETCH });
+    const nameSearch = search ? JSON.stringify({ name: [search] }) : search;
```

Comment:
Very minor suggestion: since we have the same logic in line 246, we could extract it into a small function